src/WTG.Analyzers/Analyzers/Visibility/VisibilityAnalyzer.cs

```diff
@@ -47,6 +47,7 @@ static void Analyze(SyntaxNodeAnalysisContext context, FileDetailCache cache)
 
 					case SyntaxKind.ProtectedKeyword:
 					case SyntaxKind.PublicKeyword:
+					case SyntaxKind.PartialKeyword:
```

Comment:
I also noticed that the `IsTopLevel` method near the bottom of this file is not `FileScopedNamespace`-aware. I suppose I should create another issue for that :)